browser/robots.txt

```diff
@@ -0,0 +1,11 @@
+User-Agent: *
+Disallow: *
```

Comment:
We might want to allow just the home page/root URL, which I think would be done by:

`Allow: /$`